src/patternfly/components/DataList/data-list.scss

```diff
@@ -71,41 +73,47 @@
   --pf-c-data-list--cell--WordBreak: normal;
   --pf-c-data-list--cell--m-truncate--MinWidth: 5ch;
 
-  // Toggle
+  // toggle
   --pf-c-data-list__toggle--MarginLeft: calc(var(--pf-global--spacer--sm) * -1); // offset toggle to align left
   --pf-c-data-list__toggle--MarginTop: calc(var(--pf-global--spacer--form-element) * -1);
   --pf-c-data-list__toggle-icon--Transition: .2s ease-in 0s;
   --pf-c-data-list__item--m-expanded__toggle-icon--Rotate: 90deg;
 
-  // Draggable button/icon
+  // draggable button/icon
   --pf-c-data-list__item-draggable-button--PaddingLeft: var(--pf-global--spacer--md);
   --pf-c-data-list__item-draggable-button--PaddingRight: var(--pf-global--spacer--md);
-  --pf-c-data-list__item-draggable-button--MarginTop: calc(var(--pf-global--spacer--sm) * -1);
+  --pf-c-data-list__item-draggable-button--MarginTop: 0;
   --pf-c-data-list__item-draggable-button--MarginLeft: calc(var(--pf-global--spacer--md) * -1);
```

Comment:
After looking at this again, I believe we're going to need the draggable button to have the target area of a button. Currently it isn't necessary since you can drag from anywhere in a draggable row and it will drag the row, but I believe we're going to update that so that you have to click on/drag the button.

Can you add a padding-top/bottom using the form-element spacer, and use a negative top/bottom margin (of it's top/bottom padding), and the button still be aligned? This will also mean the element should not be `display: flex` if possible, so that the line height of the button can make the button as big as it needs to be. Though you could continue to make the draggable-button-icon have a fixed width of the font-size * line-height, I just wonder if that's necessary.

Reply:
![Screen Shot 2020-12-16 at 9 43 37 AM](https://user-images.githubusercontent.com/5385435/102363219-3658ff80-3f83-11eb-9d94-7d7d3c190fd0.png)

I was able to achieve this while still using `flex`.

Reply:
@mcarrano @mceledonia wdyt about this? Or would you prefer the button only take up the space it needs, like this? 
<img width="263" alt="Screen Shot 2020-12-16 at 9 21 48 AM" src="https://user-images.githubusercontent.com/35148959/102368143-27248280-3f80-11eb-9fde-bbf1d4b92bae.png">

Reply:
I like the bigger target, personally.